File: python/99 Threads.py
import threading

def hello():
    for _ in range(11):
        print(f"Hello from thread: {threading.current_thread().name}")

t1 = threading.Thread(target=hello, name="Thread-1")
t2 = threading.Thread(target=hello, name="Thread-2")

t1.start()
t2.start()

t1.join()
t2.join()

File: java/99 Threads.java
package code;
class HelloThread extends Thread {
    @Override
    public void run() {
        for (int i = 0; i < 10; i++) {
            System.out.println("Hello from "+Thread.currentThread().getName());
        }
    }
}

public class Test {
	public static void main(String[] args) {
		HelloThread t1 = new HelloThread();
        HelloThread t2 = new HelloThread();

        t1.start();
        t2.start();
	}
}